modules/running-insights-operator-gather.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: PROCEDURE
+[id="running-insights-operator-gather_{context}"]
+= Running an Insights Operator gather operation
+
+You can run Insights Operator data gather operations on-demand. The following procedure describes how to run the default list of gather operations excluding the `workloads` operation. You can customize the on-demand gather function to exclude any gather operations you choose. If you have previously disabled Insights Operator gather operations in your cluster, this procedure will override those parameters.  
+
+:FeatureName: The `DataGather` custom resource 
+include::snippets/technology-preview.adoc[] 
+
+[NOTE]
+====
+If you enable Technology Preview in your cluster, Insights Operator runs gather operations in individual pods. This is part of the Technology Preview feature set for the Insights Operator and supports the new data gathering features.
+====
+
+.Prerequisites
```

Comment:
Should these prereqs include "You have the oc CLI installed on your cluster" for the CLI method?